### ISSUE TITLE
JS error from particles.js

### ISSUE THREAD

**Issue by doaa-altarawy** [MolSSI/QCArchiveWebsite#58]
When visiting any page other than the homepage, the following error occurs (see browser console):

```javascript

particles.min.js:9 Uncaught TypeError: Cannot read property 'getElementsByClassName' of null
    at window.particlesJS (particles.min.js:9)
    at XMLHttpRequest.i.onreadystatechange (particles.min.js:9)

```

**Comment by dgasmith** [comment]
@ishunko

**Comment by doaa-altarawy** [comment]
Yes, it is in master. It's fixed in PR #57.
